api/src/main/java/io/jsonwebtoken/lang/Classes.java

```diff
@@ -193,10 +196,18 @@ public static boolean isAvailable(String fullyQualifiedClassName) {
     public static <T> T invokeStatic(String fqcn, String methodName, Class[] argTypes, Object... args) {
         try {
             Class clazz = Classes.forName(fqcn);
-            Method method = clazz.getDeclaredMethod(methodName, argTypes);
-            method.setAccessible(true);
+            final Method method = clazz.getDeclaredMethod(methodName, argTypes);
+
+            AccessController.doPrivileged(new PrivilegedAction<Object>() {
+                @Override
+                public Object run() {
+                    method.setAccessible(true);
+                    return null;
+                }
+            });
+
             return(T)method.invoke(null, args);
-        } catch (Exception e) {
+        } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
```

Comment:
This is not backwards-compatible functionality, we can't really do this until 1.0, right?  If any other RuntimeException is thrown, it won't be caught (like it is today).

Reply:
Technically, but I think this one might be a far edge case, a RuntimeException would have to be thrown, and the caller would need to be catching an `IllegalStateException`.

Reply:
True - but it is possible given the number of projects that use jjwt.  I'd feel safer if we don't change this until 1.0.

Reply:
Could you add a //TODO on the three places and add a follow-up GitHub issue for us to remove it for 1.0 (and add it to the 1.0 release tag?)